jakttest/README.md

```diff
@@ -3,6 +3,18 @@
 Jakttest is the test driver we use to test the Jakt compiler with code snippets.
 It's also written in Jakt.
 
+## Build dependencies
+
+To build Jakttest you should use the `ninja` build system which will use the provided `build.ninja` file.
+It needs the same `clang++` that is required to build any other Jakt file, and `python` 3.6+ to run the
```

Comment:
Can you change `python` in the ninja file to `python3`? Ubuntu 22.04 doesn't have the `python` alias by default.